src/Lexicographics.dfy

```diff
@@ -0,0 +1,95 @@
+// RUN: %dafny /compile:0 "%s"
+
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT 
+*******************************************************************************/
+
+include "Relations.dfy"
+
+module Lexicographics {
+    export 
+    reveals LexicographicByteSeqBelow, LexicographicByteSeqBelowAux 
+    provides Helpers, UInt, Relations
+
+    import Helpers = Relations.Helpers
+    import opened UInt = Helpers.UInt
+    import opened Relations
+
+  // reflexivelexicographical comparison of byte sequences
+  predicate method LexicographicByteSeqBelow(x: seq<uint8>, y: seq<uint8>) 
```

Comment:
Remove "Lexicographic" from all the predicate, lemmas and method names unless strictly necessary, since you are in the Lexicographics module.